test/mason/publish-tests/badDry.chpl

```diff
@@ -6,7 +6,7 @@ use MasonNew;
 const dir = here.cwd();
 
 proc main() {
-  masonNew(['mason', 'new', 'publishCheck']);
+  masonNew(['new', 'publishCheck','']);
```

Comment:
I think we can revert these back to how they were now, unless I misunderstand the reason for changing these.